### DISCUSSION TITLE
Auto Run

### DISCUSSION THREAD

**Discussion by brentfamily51** [itsToggle/plex_debrid#97] [Q&A] [answered]
Is there a way for the python app to auto run/start?    Or does this need to be manually done?   I could run a script to get it started, but it appears to need the input of 1 to put it in run mode, so it doesn't actually start running.  Any help is appreciated.

**Comment by itsToggle** [comment, marked as answer]
You can disable the setting '/Settings/UI Settings/show menu on startup' :)

**Reply by brentfamily51** [reply]
I guess I should have tried that, I saw that setting, but it appeared "off" initially so I turned it on, figuring it might do what I wanted.   and didn't think to go back to it.  Thank you!

**Reply by itsToggle** [reply]
Nice! Ill check the default settings again, perhaps I messed up :)
